web3signer/entrypoint.sh

```diff
@@ -92,7 +92,6 @@ exec /opt/web3signer/bin/web3signer \
   --http-listen-host=0.0.0.0 \
   --http-host-allowlist="web3signer.web3signer-prater.dappnode,ui.web3signer-prater.dappnode,prysm.migration-prater.dappnode,$ETH2_CLIENT_DNS" \
   --http-cors-origins=* \
-  --metrics-enabled=true \
```

Comment:
Should not be used the env `WEB3SIGNER_METRICS_ENABLED` as the value in the flag `--metrics-enabled`?

Reply:
W3S should be configurable using environment variables as well: https://docs.eth2signer.consensys.net/en/latest/Reference/CLI/CLI-Syntax/#metrics-enabled

Reply:
I see, but this kind of breaks with the way of working we have with `EXTRA_OPTS`. Also you did not include any comment so its hard to know how it works